src/status_im/ui/screens/offline_messaging_settings/events.cljs

```diff
@@ -16,6 +18,53 @@
                               {:dispatch [:logout]}
                               (accounts-events/update-settings (assoc-in settings [:wnode network] wnode))))))
 
+(handlers/register-handler-fx
+ :save-transaction
+ (fn [{:keys [db] :as cofx} [_ wnode hash]]
+   (let [network     (ethereum/network->chain-keyword (:network db))
+         settings    (get-in db [:account/account :settings])
+         transaction @(subscribe [:wallet.transactions/transaction-details])]
```

Comment:
it's not ideologically right to use subscriptions in events, you need to retrieve this data from db